background/index.ts

```diff
@@ -82,6 +86,79 @@ async function main() {
     },
     ["userOpPool"]
   )
+
+  const fetchData = async () => {
+    const fetchUserOpsSent = async () => {
+      const timeout = 1500
+      console.log(`[background] fetch userOp sent every ${timeout} ms`)
+
+      const s = storage.get()
+      const nm = new NetworkStorageManager(storage)
```

Comment:
Why not reuse the one in the main function?

Reply:
After testing,
it is necessary here to retrieve the latest state, and establish the latest NetworkStorageManager from local storage every 1.5 seconds;
otherwise, the obtained userOp will always be {} empty.

Reply:
After testing,
The `NetworkStorageManager` can reuse the declared `networkManager`,
but the `storage.get()` needs to be refreshed every 1.5 seconds; otherwise, the `state` variable `userOpPool` remains `{}` empty.